src/campaign/components/ItemModal/__test__/ItemForm.test.tsx

```diff
@@ -1,3 +1,4 @@
+import userEvent from "@testing-library/user-event";
```

Comment:
How long do these tests take to run using userEvent vs fireEvent?

Reply:
The tests in this file take ~100ms at the slowest. userEvent is super useful for typing text. I don't even know how you'd achieve that with fireEvent. Probably a series of `.keyPress` calls for each character.